modules/rds/secrect.tf

```diff
@@ -6,6 +6,3 @@ data "aws_secretsmanager_secret_version" "database-password" {
   secret_id = data.aws_secretsmanager_secret.database-dev.id
```

Comment:
I will suggest you handle this secret outside of the rds module and then pass in the value into the rds module